app/player/player.controller.js

```diff
@@ -3,12 +3,15 @@
 angular.module('app.player')
 
 .controller('playerCtrl', ['$localStorage', function($localStorage) {
+	var addButtonClicked = false;
 	var vm = this;
 	//Array with all players ready to start the game
-	vm.players = [];
+	//vm.players = [];
```

Comment:
A comment should be used for explain what the behaviour of the code, not to hide it